src/Goerli-DssSpell.t.base.sol

```diff
@@ -562,7 +562,7 @@ contract GoerliDssSpellTestBase is Config, DSTest, DSMath {
                     assertTrue(mat >= RAY && mat <= 300 * RAY, concat("TestError/vat-mat-range-", ilk));  // cr gt 100% and lt 30000%
                 } else {
                     assertEq(mat, normalizedTestMat, concat("TestError/vat-mat-", ilk));
-                    assertTrue(mat >= RAY && mat < 10 * RAY, concat("TestError/vat-mat-range-", ilk));    // cr gt 100% and lt 1000%
+                    assertTrue(mat >= RAY && mat <= 50 * RAY, concat("TestError/vat-mat-range-", ilk));    // cr gt 100% and lt 5000%
```

Comment:
this shouldn't be changed as it refers to normal ilk `mat` value range, I would recommend to add another block or condition in the above block, used for the lerp.

Reply:
How this condition should look like? Just check specifically for `RENBTC-A`

Reply:
there is already a block before with an higher `mat` range, you could add an or condition for the specific ilk there

Reply:
Done